src/main/java/pages/Homepage.java

```diff
@@ -12,8 +15,36 @@ public class Homepage {
     private final By welcomeText = By.className("logged-in");
     private final By mainBannerTopText = By.cssSelector(".blocks-promo .home-main .info");
     private final By mainBannerMainText = By.cssSelector(".blocks-promo .home-main .title");
-    private final By mainBannerButtonText = By.cssSelector(".action.more.button");
+    private final By mainBannerButton = By.cssSelector(".action.more.button");
     private final By mainBanner = By.cssSelector(".block-promo.home-main");
+    private final By mainBannerLink = By.cssSelector(".blocks-promo > a");
+    private final By mainBannerImageSource = By.cssSelector(".blocks-promo > a img");
+    private final By pantsBanner = By.cssSelector(".block-promo.home-pants");
+    private final By pantsBannerTopText = By.cssSelector(".home-pants .content .title");
+    private final By pantsBannerMainText = By.cssSelector(".home-pants .content .info");
+    private final By pantsBannerBottomText = By.cssSelector(".home-pants .content .action");
+    private final By pantsBannerImageSource = By.cssSelector(".block-promo.home-pants >img");
```

Comment:
It's not ImageSource -> It's only Image.
Also please remember about space after '>' in selector.